### PR TITLE
Added project files and updated repository URL

### PR DESCRIPTION
**Pull request by Vr978**
Purpose:
This PR introduces the project files from [my original repository](https://github.com/Vr978/DAYTONA-ASKPIXIE.git) into the sample repository.
Additionally, the repository URL has been updated from the original project repository to point to https://github.com/daytonaio/sample-python-ask-pixie, as per the instructions.

Summary of Changes:

Added all project files and folders to the target repository.
Updated the URL in all instances where the project repository is referenced.
Testing and Validation:
Changes have been validated locally to ensure consistency and functionality after the update.

Relevant Links:

Original Repository: [Vr978/DAYTONA-ASKPIXIE](https://github.com/Vr978/DAYTONA-ASKPIXIE.git)
Target Repository: [daytonaio/sample-python-ask-pixie](https://github.com/daytonaio/sample-python-ask-pixie)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,82 @@
...
+
+### Open Using Daytona  
+
+1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).
+  
+2. **Create the Workspace**: (by adding git provider)
    ```bash  
-   command to start the app
-   ```  
+   daytona create https://github.com/Vr978/sample-python-ask-pixie.git --devcontainer-path=.devcontainer/devcontainer.json
```
Comment (lbrecic) [inline README.md]: Change the repo url to correct one

**Review by lbrecic — CHANGES_REQUESTED** [review]
Also, DCO is failing, tend to that please

**Comment by lbrecic** [comment]
@Vr978 why did you close this?

File: README.md (hunk 1 above)
Comment (Vr978) [reply README.md]: oh extremely sorry for that!!! I think the dco problem must be resolved now and i am changing the reamde.md to: https://github.com/daytonaio/sample-python-ask-pixie.git

**Pull request closed without merging**